### PR TITLE
Nodejs-week3 warmup initial commit

### PR DESCRIPTION
**Pull request by agkabir**

### CODE REVIEW

File: nodejs/week3/app.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+import express from "express";
+const app = express();
+const port = process.env.PORT;
```
Comment (itsgauravsaxena) [inline nodejs/week3/app.js:5]: You should add PORT in .env file else port will be undefined

File: nodejs/week3/.env.example [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Database - change to fit your database
+DB_HOST = 127.0.0.1 
+DB_USER = root
+DB_PASSWORD = db_password
+DB_NAME = db_name
+DB_PORT = db_port
```
Comment (itsgauravsaxena) [inline nodejs/week3/.env.example]: Missing PORT

**Review by itsgauravsaxena — CHANGES_REQUESTED** [review]
Did you try sql injection demo and record it?

File: nodejs/week3/app.js (hunk 1 above)
Comment (agkabir) [reply nodejs/week3/app.js:5]: I have PORT defined in .env file. By mistake I forgot to write it on .env.example

File: nodejs/week3/.env.example (hunk 2 above)
Comment (agkabir) [reply nodejs/week3/.env.example]: Yes. you are right. but I have it in .env

**Pull request merged**